webhooks-extension/docs/GettingStarted.md

```diff
@@ -0,0 +1,161 @@
+
+### Create credentials: Git
+
+You will need to create a secret for Git if any of your GitHub repositories require authentication in order to clone them. This is true for GitHub private repositories, and for many GitHub Enterprise installations. While Tekton supports SSH and Basic authentication for Git, only the latter is supported by the webhooks extension today. Basic authentication can be used to provide either username/password, or an access token. Access tokens are our recommended approach. 
+
+Create an access token via GitHub. As of August 2019, for a given GitHub repository, visit [your name icon in the top right corner] > Settings > Developer settings > Personal access tokens > Generate new token and create a token with `repo` permissions. Add `admin:repo_hook` permissions if you are going to use the same access token to create webhooks. 
+
+Store the access token in a Kubernetes secret via the Tekton Dashboard's 'Secrets' menu. This panel will help you add the right annotation to the secret, and patch it onto the specified service account. Annotations should typically be of the form, `tekton.dev/git-0: https://github.com`. If you are using an access token, store it in the 'password' field. Be sure to specify the service account that will be used by the associated Tekton pipeline. 
+
+![Creating a Git secret](createGitSecret.png?raw=true "Creating a Git secret")
+
+### Create credentials: Docker
+
+Credentials are normally required to push images to a docker registry. Some docker registries support access tokens. Others such as docker hub only support username/password. If you are pushing images to Docker Hub the annotation should be of the form, `tekton.dev/docker-0: https://index.docker.io/v1/`. Again, select the service account of the associated Tekton pipeline so that it is correctly patched with the new docker credentials. 
+
+### Import pipeline definitions
+
+A Tekton Pipeline is composed of one or more Tasks. We expect that most users will store their Pipeline and Task definitions in git. You may wish to `git clone` and `kubectl apply` these definitions manually, or more likely, as part of an automated installation and setup. For your convenience, the dashboard provides a menu option, 'Import Tekton resources'. 
+
+![Import pipeline definitions](importPipelines.png?raw=true "Importing pipeline definitions")
+
+This screen shot shows a user importing pipeline definitions from a git repository. This panel will drive the Pipeline `pipeline0` 
+
+
+## Creating a new webhook
+
+The Tekton dashboard webhooks extension creates an association between a git repository and a Tekton pipeline. Changes to a branch or pull request in the associated git repository will cause a new PipelineRun and PipelineResource objects to be created. This section covers the various setup and configuration options in more detail. The image below shows the current form of the main 'Create Webhook' panel in the dashboard. 
```

Comment:
Git* everywhere instead of `git`

`cause a new PipelineRun`, -> `trigger`? We can be a little fancier and probably translates better

I'm hesitant about including "current form" references as this implies a need to update this document whenever we make a change, but your screenshot is good. Can you rename it please to createWebhook.png instead of screenShot1.png?

Reply:
done